### PR TITLE
IOW-641 restore prod aurora cluster from snapshot to test tier

### PR DESCRIPTION
**Pull request by kkehl-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
For support of load testing, create a state machine to:

1. restore a production aurora cluster snapshot to the test tier
2. change the password on the restored cluster
3. create a db instance
4. delete the db instance
5. delete the cluster

Description
-----------

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by kkehl-usgs** [comment]
Prior to Jenkins breaking this afternoon, I did see an aurora cluster get restored and was able to log into it in pgadmin.

There are no tests because this service is the "load test" service.  I'm not sure how much value there is in testing a test, when there is no code here except making API calls to AWS and everything is going to be mocked anyway.

The commented out code is for other stories like IOW-642 and IOW-646, but feel free to call out if something doesn't make sense in term of the order of operations or anything else.

I'm thinking that aqts-capture-trigger will read the bucket name from the secrets manager, and that this service will falsify the secrets related to the database user name and password, so most of the lambdas won't need any changes.

File: serverless.yml [hunk 1]
```diff
@@ -26,25 +26,254 @@ custom:
...
+
+
+
+
+
+
+
+
+
+
```
Comment (ssoper-usgs) [inline serverless.yml]: I think you made a note about keeping the below commented out code, but feel free to remove the above whitespace.

File: serverless.yml [hunk 2]
```diff
@@ -26,25 +26,254 @@ custom:
...
```
Comment (ssoper-usgs) [inline serverless.yml]: Same comment here.

**Review by ssoper-usgs — APPROVED** [review]

**Comment by kkehl-usgs** [comment]
@ayan-usgs I made the changes you requested.

**Pull request merged**